spec/views/accounts/_identity_verification.html.erb_spec.rb

```diff
@@ -561,4 +573,71 @@
       end
     end
   end
+
+  describe 'connect to SP alert' do
+    let(:post_idv_follow_up_url) { 'https://example.com/followup' }
+    let(:initiating_service_provider) do
+      build(
+        :service_provider,
+        friendly_name: initiating_sp_name,
+        post_idv_follow_up_url:,
+        return_to_sp_url: nil,
+      )
+    end
+    let(:initiating_sp_name) { 'Test SP' }
+    let(:user) { create(:user, identities: [identity].compact, profiles: [profile].compact) }
+    let(:profile) do
+      build(:profile, :active, initiating_service_provider:)
+    end
+    let(:last_ial2_authenticated_at) { nil }
+    let(:identity) do
+      build(
+        :service_provider_identity,
+        service_provider: initiating_service_provider.issuer,
+        last_ial2_authenticated_at:,
+      )
+    end
+
+    context 'with a user who has not connected to their initiating service provider' do
+      context 'the serive provider has a post-idv follow-up url' do
+        it 'renders an alert to connect to IdV SP with a link' do
+          expect(rendered).to have_content(
+            t('account.index.verification.connect_idv_account.intro'),
+          )
+          expect(rendered).to have_link(
+            t('account.index.verification.connect_idv_account.cta'),
+            href: post_idv_follow_up_url,
+          )
+        end
+      end
+
+      context 'the serive provider does not have a post-idv follow-up url' do
```

Comment:
Nit: `service` instead of `serive`
```suggestion
      context 'the service provider does not have a post-idv follow-up url' do
```